Warcraft.NET/Files/ADT/Chunks/Legion/MLLV.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.IO;
+using System.Numerics;
+using Warcraft.NET.Extensions;
+using Warcraft.NET.Files.Interfaces;
+
+namespace Warcraft.NET.Files.ADT.Chunks.Legion
+{
+    /// <summary>
+    /// MLLV Chunk - Level of detail vertices
+    /// </summary>
+    public class MLLV : IIFFChunk, IBinarySerializable
+    {
+        /// <summary>
+        /// Holds the binary chunk signature.
+        /// </summary>
+        public const string Signature = "MLLV";
+
+        /// <summary>
+        /// Unknown.
+        /// </summary>
+        public Vector3[] LiquidVertices { get; set; }
+
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MLLI"/> class.
+        /// </summary>
+        public MLLV()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MLLV"/> class.
+        /// </summary>
+        /// <param name="inData">ExtendedData.</param>
+        public MLLV(byte[] inData)
+        {
+            LoadBinaryData(inData);
+        }
+
+        /// <inheritdoc/>
+        public void LoadBinaryData(byte[] inData)
+        {
+            using (var ms = new MemoryStream(inData))
+            using (var br = new BinaryReader(ms))
+            {
+                var vertexCount = inData.Length / 12;
+                LiquidVertices = new Vector3[vertexCount];
+                for (var i = 0; i < vertexCount; ++i)
+                {
+                    LiquidVertices[i] = br.ReadVector3(Structures.AxisConfiguration.Native);
```

Comment:
Maybe remove the Structures.AxisConfiguration.Native and use WriteVector3 for writing